pkg/authorization/api.go

```diff
@@ -39,6 +41,9 @@ type Request struct {
 
 	// ResponseHeaders stores the response headers sent to the docker daemon
 	ResponseHeaders map[string]string `json:"ResponseHeaders,omitempty"`
+
+	// Policies stores a list of policies associated/identified with given request till this point
+	Policies map[container.PolicyType]string `json:"Policies,omitempty"`
```

Comment:
the authz plugins can be chained, so I thought it might be better to pass along the policy as it is computed to all plugins in the chain along with rest of the request body.